ipatests/test_integration/test_user_permissions.py

```diff
@@ -190,6 +190,36 @@ def test_user_add_withradius(self):
              '--user-auth-type', 'radius', '--radius-username', 'testradius',
              'testradius', '--radius', radiusproxy])
 
+    def test_user_add_fix_capitalization_error(self):
+        """
+        Test that an existing user which has been modified using ipa user-mod
+        and has the same first and last name beginning with caps does not
+        throw the error 'ipa: ERROR: Type or value exists:' and i
+        instead gets modified
+
+        This is a test case for Pagure issue
+        https://pagure.io/freeipa/issue/5879
+        """
+        # kinit admin
+        tasks.kinit_admin(self.master)
+
+        # Create an ipa-user
+        ipauser = 'ipauser1'
+        first = 'ipauser'
+        modfirst = 'Ipauser'
```

Comment:
small nitpick, it'd be nicer if modfirst was really different from first, like IpaUser. As it stands the capitalized "I" is not the easiest change to pick up when reading the test results.